payment/utils.py

```diff
@@ -67,6 +69,22 @@ def send_confirmation_email(user, order_number):
     email.attach_alternative(html_content, "text/html")
     email.send()
 
+def register_user_verified_cohort(user, order):
+    course = get_course(order)
+    course_key = CourseKey.from_string(course.key)
+    storedcourse = modulestore().get_course(course_key)
+    verified_cohort_name = getattr(settings, 'DEFAULT_VERIFIED_COHORT_NAME',
+                                   "certificat")
+    verified_cohorts = [c for c in get_course_cohorts(storedcourse)
```

Comment:
I understand that you want to add every cohort that as "certificat" in its name. This seems fragile and tied to a language.
Can we base it on a cohort id or machine name?

Also, adding one cohort seems enough, why add several?

Reply:
> I understand that you want to add every cohort that has "certificat" in its name. This seems fragile and tied to a language. Can we base it on a cohort id or machine name?

Cohorts are specified per course. We could define a `VERIFIED_COHORT_IDS` setting, which would store a list of cohort IDs to which users would be added, but then we would have to specify different cohorts for every course. Is this the desired behaviour @laurentdavid? If yes, then I suggest we list the cohort names, and not the IDs. Something along the lines of:

```
VERIFIED_COHORT_NAMES = {
  "edX/toy/2012_Fall": ["Cohorte certifiée"]
}
...
for cohort in get_course_cohorts(storedcourse):
  if cohort.name in settings.DEFAULT_VERIFIED_COHORT_NAME.get(course.key, []):
    add_user_to_cohort(cohort, user.email)
```

@sampaccoud would this be acceptable?

> Also, adding one cohort seems enough, why add several?

@laurentdavid if we don't need multiple cohorts, I suggest we stick with one verified cohort per course.

Reply:
@regisb I think we don't want to have to modify settings each time a new course has to use this feature. I was wondering if there was a better field on the cohort on which we could look for a pattern.